Source/Core/Common/SymbolDB.h

```diff
@@ -37,6 +37,9 @@ struct Symbol
     Data,
   };
 
+  Symbol() = default;
+  explicit Symbol(const std::string& name) { Rename(name); }
```

Comment:
Where is this used?  It's marked as explicit but missing it..

Was wondering if it made sense to make it a move constructor by introducing a new `Rename`.  But if we can't move, no worries.

Reply:
>  It's marked as explicit but missing it..

I'm not sure what this means

The constructor itself is invoked in one of the emplace calls (which forwards into it), allowing the Symbol instance to be constructed directly in the container without needing to move into it.

Reply:
I'm assuming you mean in `AddKnownSymbol`.  Sorry, not super familiar with this code (first time I've looked at it).  Looks like `name` is also `const&` so we can't move it.  That's what I assumed but wanted to check.